packages/core/package.json

```diff
@@ -13,10 +45,17 @@
     "access": "public"
   },
   "files": [
-    "lib",
+    "src",
     "types",
-    "*.js"
+    "*.js",
+    "dist"
   ],
+  "scripts": {
+    "size": "../../bin/size dist/bugsnag.min.js",
+    "clean": "rm -fr dist && mkdir dist",
+    "build": "npm run clean && npm run build:npm",
+    "build:npm": "rollup --config rollup.config.npm.mjs"
+  },
```

Comment:
TypeScript compilation is added (with support for .js files) to the build step but the dist assets are not yet being used